lib/src/__tests__/DateTimePickerTestingLib.test.tsx

```diff
@@ -63,4 +63,41 @@ describe('<DateTimePicker />', () => {
     await waitFor(() => screen.getByRole('dialog'));
     expect(screen.getByLabelText('11 hours').getAttribute('aria-disabled')).toBe('true');
   });
+
+  it('prop: views – seconds is visible', async () => {
+    render(
+      <MobileDateTimePicker
+        open
+        onChange={() => {}}
+        ampm={false}
+        renderInput={props => <TextField {...props} />}
+        value={utilsToUse.date('2018-01-02T03:04:05.000Z')}
+        views={['year', 'month', 'date', 'hours', 'minutes', 'seconds']}
+      />
+    );
+
+    await waitFor(() => screen.getByRole('dialog'));
+    expect(getByMuiTest('seconds').firstChild?.textContent).toBe('05');
```

Comment:
```suggestion
    expect(getByMuiTest('seconds')).to.have.text('05');
```

Reply:
I don't think this is valid

I get a type error with this change. `Property 'to' does not exist on type 'JestMatchersShape  , Matchers  , string | null>>'.`

Reply:
Oh right, It's probably because we miss

https://github.com/mui-org/material-ui/blob/d1287f1874dffeade600364b36727fa7878e9a75/test/utils/initMatchers.js#L16

Reply:
Ok, forget about it, we will have it resolved by merging the repository in the main one.